### ISSUE TITLE
Duplicated account issues on the Product Feed page

### ISSUE THREAD

**Issue by eason9487** [woocommerce/google-listings-and-ads#1198]
### Describe the bug:

Appears 8 duplicated account issues on the Product Feed page.

I also check the API via [the "Try this API" function on Google website](https://developers.google.com/shopping-content/reference/rest/v2.1/accountstatuses/get). And it results in 8 **almost duplicated** account issues. The different parts are the `country` and `destination` properties.

https://github.com/woocommerce/google-listings-and-ads/blob/fe1bbf4d49b6a7d53a38403ae17bc6a807087fd8/tests/proxy/mocks/mc/content/accountstatuses.json#L6-L78

### Steps to reproduce:

💡  Since I didn't go further to survey the reasons why Google API returns such content, and this issue might not be reproducible for every MC account, I use the API response I got as the mock data.

1. Checkout `mock/1198-reproduce-duplicated-mc-account-issues` branch.
2. Follow [the proxy instructions](https://github.com/woocommerce/google-listings-and-ads/blob/fe1bbf4d49b6a7d53a38403ae17bc6a807087fd8/tests/proxy/proxy.md) to start the local proxy server.
3. Delete the transient of MC issues `wp transient delete gla_mc_statuses`.
4. Go to the Product Feed page.

### Expected behavior:

Not sure what's the expected behavior. But at least, it should not show totally the same and duplicated account issues in UI. Probably we could either show only one issue for almost the same issues or show their differences in UI.

- [x] Delete `mock/1198-reproduce-duplicated-mc-account-issues` branch after resolving this issue.

### Actual behavior:

![image](https://user-images.githubusercontent.com/17420811/149749818-b81ed1db-b156-49e4-ae83-de344ddc453a.png)

![image](https://user-images.githubusercontent.com/17420811/149749910-826f890c-604f-48f0-9d5b-285d834daa01.png)

### Additional details:

Looks like it has some relation to the target audience countries.

A few days ago, I finished onboarding setup with 'US', 'JP', 'AU' and 'TW' as target audience countries. From the raw response data, the `country` properties are also these four countries.

**Comment by findingsimple** [comment]
@j111q could you remind us (i'm feeling rusty) how we should be presenting issues that are the same but for different target countries.

**Comment by j111q** [comment]
Thanks for flagging this one @eason9487 .

> we could either show only one issue for almost the same issues

This 👆 was what we intended to happen, I believe. So if it's the same issue for multiple countries, we only show it once. The reason was because we didn't want to spam the user with so many issues for multiple countries, when typically we know that if it's an issue in one country, it's probably going to be the same in other countries. 

LMK what y'all think though!

**Comment by eason9487** [comment]
Thanks for replying @j111q .

> So if it's the same issue for multiple countries, we only show it once. The reason was because we didn't want to spam the user with so many issues for multiple countries, when typically we know that if it's an issue in one country, it's probably going to be the same in other countries.

I looked at Google Merchant Center and they're also merged as one issue item. If we also use a consistent way, it'd be less confusing for users when they check on both admin websites. Also, we could consider further adding a similar expandable UI to prompt users if needed in the future.

![Kapture 2022-01-25 at 18 31 30](https://user-images.githubusercontent.com/17420811/150960479-3524764a-b18a-4825-8750-70dc24fff52b.gif)

**Comment by mikkamp** [comment]
Thanks for the report, I'm able to reproduce the same results. Although from my recollection this wasn't the case when we created the code for it.

When we created the code for retrieving product issues we explicitly check if the issue description has already been recorded and we then merge the list of countries. This can be seen [here](https://github.com/woocommerce/google-listings-and-ads/blob/1.11.1/src/MerchantCenter/MerchantStatuses.php#L443-L446), we also save that in the database table `wp_gla_merchant_issues` in the column `applicable_countries`.

However for account issues we don't combine per country, so they get added multiple times to the database (as far as I recall it wasn't necessary when we first implemented it).

Another thing I noticed is that the error can be repeated for a different destination:

```json
{
    "id": "custom_label",
    "title": "Account suspended due to policy violation: Misrepresentation",
    "country": "NL",
    "severity": "critical",
    "documentation": "https://support.google.com/merchants/answer/6150127",
    "destination": "Shopping"
},
{
    "id": "custom_label",
    "title": "Account suspended due to policy violation: Misrepresentation",
    "country": "NL",
    "severity": "critical",
    "documentation": "https://support.google.com/merchants/answer/6150127",
    "destination": "SurfacesAcrossGoogle"
}
```

According to the [documentation](https://developers.google.com/shopping-content/reference/rest/v2.1/accountstatuses/get#query-parameters) if we don't set the destination parameter it should only return Shopping.

> If set, only issues for the specified destinations are returned, otherwise only issues for the Shopping destination.

Will need to clarify that with Google as that's not the result we are seeing, and it's adding to the duplicate account issues.
